spring-rabbit/src/main/java/org/springframework/amqp/rabbit/support/PublisherCallbackChannel.java

```diff
@@ -86,10 +84,10 @@
 	/**
 	 * Set a callback to be invoked after the ack/nack has been handled.
 	 * @param callback the callback.
-	 * @param proxy the proxy.
+	 * @param proxyForThis the proxy for this channel.
 	 * @since 2.1
 	 */
-	void setAfterAckCallback(Consumer<ChannelProxy> callback, ChannelProxy proxy);
+	void setAfterAckCallback(Consumer<Channel> callback, Channel proxyForThis);
```

Comment:
Yeah... Still confusing: the `PublisherCallbackChannel` is a `Channel` per se. 

Doesn't my idea about relationship `Map` work?